Create ui-c8y-1023-75-1-not-able-to-assign-a-different-device-profile-in-the-ui.md (#4599)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1023-75-1-not-able-to-assign-a-different-device-profile-in-the-ui.md

---
date: ""

> title: Not able to assign a different Device Profile in the UI (#11842)

Reviewer: ```suggestion
title: Resolved device profile selection reset on the Device profile tab
```

> title: Resolved device profile selection reset on the Device Profile tab

Reviewer 2: ```suggestion
title: Device profiles can properly be reassigned
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: DM-5985
version: 1023.75.1
---

> Not able to assign a different Device Profile in the UI (#11842)

Reviewer: ```suggestion
Previously, the Device profile tab prevented users from switching device profiles if one was already assigned, as the selection dropdown would automatically reset to the current device profile before the assignment could be completed. This issue has been resolved — the UI now correctly maintains the selected value, ensuring that users can successfully reassign or update profiles for any device.
```

> Previously, the Device Profile tab prevented users from switching device profiles if one was already assigned, as the selection dropdown would automatically reset to the current device profile before the assignment could be completed. This issue has been resolved. The UI now correctly maintains the selected value, ensuring that users can successfully reassign or update profiles for any device.

Reviewer 2: ```suggestion
Previously, the **Device profile** tab prevented users from switching device profiles if one profile was already assigned. The selection dropdown would automatically reset to the current device profile before the assignment could be completed. This issue has been resolved. The UI now correctly maintains the selected value, ensuring that users can successfully reassign or update profiles for any device.
```
